README.md

```diff
@@ -1,2 +1,241 @@
+  </ul>
+</details> -->
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.enterprisedb.com/products-services-training/pgdownload">Postgresql Client for Windows</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.enterprisedb.com/products-services-training/pgdownload">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Patients table]**
+- **[Invoices table]**
+- **[Treatments table]**
+- **[Invoice items table]**
+- **[medical histories table]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- > Add a link to your deployed project.
+
+- [Live Demo Link](<replace-with-your-deployment-URL>)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> This projects provides basic structure to manage and store datas about patients.The data model is extensible.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: 
+
+```sh
+  Postgresql Client for windows installed
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:patriciachrysy/clinic-database.git
+```
+
+<!--### Install
+
+Install this project with:
+
+
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  psql -U postgres
+```
+Then copy and paste the content of schema_based_on_diagram.sql file.
+<!--### Run tests
+
+To run tests, run the following command:
+
+
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+
+<!--### Deployment
+
+You can deploy this project using:
+
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> These are the people who worked on this project.
+
+👤 **Manezeu Patricia**
+
+- GitHub: [@githubhandle](https://github.com/patriciachrysy)
+- Twitter: [@twitterhandle](https://github.com/patriciachrysy)
+- LinkedIn: [LinkedIn](https://github.com/patriciachrysy)
+  
+
+👤 **Al Amin Khan Shakil**
+
+- GitHub: [Al Amin Khan Shakil](https://github.com/Al-Amin-Khan-Shakil)
+- Twitter: [Al Amin Khan Shakil](https://twitter.com/AlAminKhan85004)
+- LinkedIn: [Al Amin Khan Shakil](https://www.linkedin.com/in/al-amin-khan-shakil/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
- [ ] Hey there! 📄 It's a great practice to have a clear documentation of future features in your README 🚀. This not only keeps fellow developers informed about what's in the pipeline 🛠️ but also helps prevent them from raising issues that are already on your radar 🚧. This way, everyone is on the same page and can contribute more effectively to the project's development. 🤝